Platform/ElkhartlakeBoardPkg/Library/Stage1BBoardInitLib/Stage1BBoardInitLib.c

```diff
@@ -41,6 +41,8 @@
 #include <GpioConfig.h>
 #include <Library/GpioLib.h>
 
+UINT8   mTccS0ixSupport    = 0;

```

Comment:
Normal global variable cannot be used in Stage1A/1B since it can be executed directly from flash.
Only read-only global variable is allowed.

Reply:
Noted. Done removing it and reworked.